tests/ltp/Makefile

```diff
@@ -128,12 +128,12 @@ add_host_users_and_groups:
 	sudo useradd -g grp701 -u 100703 user703
 
 del_host_users_and_groups:
-	sudo userdel user700
-	sudo userdel user702
-	sudo userdel user703
-	sudo groupdel grp701
-	sudo groupdel grp704
-	sudo groupdel grp705
+	sudo userdel user700 || /bin/true
+	sudo userdel user702 || /bin/true
+	sudo userdel user703 || /bin/true
```

Comment:
Can we just use `sudo userdel user700 || true`?

Reply:
Hi Xuejun, would it ever act differently?

Reply:
I think it's the same. We have a few Makefiles use `... || true`

Reply:
Gotcha, if they act the same I think its okay to keep this here, what do you think?

Reply:
Yes since this is a blocking issue. Feel free to merge after the pipeline passes.